Test/ValidationErrorsConstraint.php

```diff
@@ -11,9 +11,6 @@
 
 namespace Liip\FunctionalTestBundle\Test;
 
-// BC
-class_alias('\PHPUnit_Framework_Constraint', '\PHPUnit\Framework\Constraint\Constraint');
```

Comment:
Okay, my previous comment was a bad idea, we need that, even if I don't know why. :)